src/api/tests/orders/markAsReceived.spec.ts

```diff
@@ -0,0 +1,470 @@
+import { API_ERRORS, STATUS_CODES, TAGS } from 'data';
+import { ORDER_HISTORY_ACTIONS, ORDER_STATUSES } from 'data/orders';
+import { errorResponseSchema, orderSchema } from 'data/schemas';
+import { expect, test } from 'fixtures';
+import { IOrderFromResponse } from 'types';
+import { extractIds, generateID } from 'utils';
+import { validateResponse, validateSchema } from 'utils/validations';
+
+test.describe('[API] [Orders] [Mark Products as Received | from status In Process]', () => {
+  let token = '';
+  let order: IOrderFromResponse;
+
+  test.beforeEach(async ({ signInApiService }) => {
+    token = await signInApiService.getAuthToken();
+  });
+
+  test.afterEach(async ({ ordersApiService }) => {
+    await ordersApiService.clear(token);
+  });
+
+  test(
+    'Should mark all (1>) products as received',
+    {
+      tag: ['@001_O_MR_PUT_API', TAGS.API, TAGS.SMOKE, TAGS.REGRESSION],
+    },
+    async ({ ordersController, ordersApiService }) => {
+      order = await ordersApiService.createInProcess(token, { productCount: 5 });
+      const prodID = extractIds(order.products);
+      const response = await ordersController.receiveProduct(order._id, prodID, token);
+
+      validateResponse(response, STATUS_CODES.OK, true, null);
+      validateSchema(orderSchema, response.body);
+
+      await test.step('Validated order status is RECEIVED', () => {
+        expect.soft(response.body.Order.status).toEqual(ORDER_STATUSES.RECEIVED);
+      });
```

Comment:
`Always add an error message to expect, even for a trivial check.` - this option looks simpler than adding a full step. 
```suggestion
        expect.soft(response.body.Order.status,
          'Order status is not RECEIVED')
          .toEqual(ORDER_STATUSES.RECEIVED);

```